src/panel.css

```diff
@@ -174,6 +174,38 @@ a:hover {
   background-image: url("/img/site-remove-icon.svg");
 }
 
+.highlight-on-hover.remove-site-from-container.disabled-button {
+  cursor: not-allowed;
+  position: relative;
+  z-index: 0;
+}
+
+.highlight-on-hover.remove-site-from-container.disabled-button::after, .highlight-on-hover.remove-site-from-container.disabled-button span {
+  opacity: 0.4;
+}
+
+#button-remove-site-tooltip {
+  display: none;
+  padding: 5px 10px 10px;
+  width: 250px;
+  max-width: 250px;
+  z-index: 9999;
+  position: absolute;
+  left: 1rem;
+  bottom: 2em;
```

Comment:
Note that this differs from the InVision mockup as the panel has `overflow:hidden` on it, not allowing for the tooltip to be visible outside of its' parent container. 

InVision mockup: 
![image](https://user-images.githubusercontent.com/2692333/71696885-dd955480-2d7b-11ea-80ad-239d4b45797b.png)

cc/ @e-pang